test/server/cards/03_TWI/leaders/GeneralGrievousGeneralOfTheDroidArmies.spec.ts

```diff
@@ -0,0 +1,83 @@
+describe('General Grievous, General Of The Droid Armies', function () {
+    integration(function (contextRef) {
+        describe('General Grievous\'s leader undeployed ability', function () {
+            it('should Sentinel to a Droid unit for this phase', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['battle-droid-legion'],
+                        spaceArena: ['devastating-gunship'],
+                        leader: 'general-grievous#general-of-the-droid-armies'
+                    },
+                    player2: {
+                        groundArena: ['battlefield-marine', 'atst'],
```

Comment:
We should add an opponent Droid unit to make sure it's targetable, same for the other test case